tests/e2e/steps_partial_set_security.go

```diff
@@ -127,25 +115,15 @@ func stepsOptInChain() []Step {
 			},
 		},
 		{
-			Action: VoteGovProposalAction{
-				Chain:      ChainID("provi"),
-				From:       []ValidatorID{ValidatorID("alice"), ValidatorID("bob")},
-				Vote:       []string{"yes", "yes"},
-				PropNumber: 1,
-			},
-			State: State{
-				ChainID("provi"): ChainState{
-					Proposals: &map[uint]Proposal{
-						1: ConsumerAdditionProposal{
-							Deposit:       10000001,
-							Chain:         ChainID("consu"),
-							SpawnTime:     0,
-							InitialHeight: clienttypes.Height{RevisionNumber: 0, RevisionHeight: 1},
-							Status:        strconv.Itoa(int(gov.ProposalStatus_PROPOSAL_STATUS_PASSED)),
-						},
-					},
-				},
+			Action: UpdateConsumerChainAction{
+				Chain:         ChainID("provi"),
+				From:          ValidatorID("alice"),
+				ConsumerChain: ChainID("consu"),
+				SpawnTime:     0, // launch now
```

Comment:
Note that `providerKeeper.CanLaunch` checks that:
```
	spawnTimeIsNotZero := !initializationParameters.SpawnTime.Equal(time.Time{})
```
Not sure what this under the hoods.

Reply:
yeah SpawnTime 0 means in e2e that spawn time is  0 millisec after the test was started -- where testconfig.Now is set.
so here it means that the spawn time is in the past and the chain should launch.
agree that it's confusing especially where spwan time 0 is interpreted differently with permissionless.

Reply:
Thanks for the explanation. I guess we can leave it as is then.